runtest.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import unittest
+import subprocess as sp
+import os
+
+qemuCmd = "qemu-arm -L /usr/arm-linux-gnueabihf".split()
+amacc = "./amacc"
+gcc = "arm-linux-gnueabihf-gcc"
```

Comment:
We can make these as options to tester, allowing change directory of compiled files. However it is a little too complicated for simple unittests.